core/basicobject/basicobject_spec.rb

```diff
@@ -76,7 +76,7 @@ def test_method() :test end
     obj.instance_variable_set(:@test, :value)
     obj.instance_variable_get(:@test).should == :value
 
-    obj.send(:hash).should be_an_instance_of(Fixnum)
+    lambda { obj.send(:hash) }.should_not raise_error
```

Comment:
Would it work for Opal if it was
`obj.send(:hash).should be_an_instance_of(Integer)` ?
That would be a more precise assertion.